lib/screens/v2/candidate_generator/widgets/reconstructed_alert.ex

```diff
@@ -16,7 +16,7 @@ defmodule Screens.V2.CandidateGenerator.Widgets.ReconstructedAlert do
   Given the routes, fetch all alerts for the route
   """
   def reconstructed_alert_instances(
-        %Screen{app_params: %PreFare{header: %CurrentStopId{stop_id: stop_id}}} = config,
+        %Screen{app_params: %PreFare{alert_widget: %CurrentStopId{stop_id: stop_id}}} = config,
```

Comment:
Depending on your answer to my question above, should change these to include reconstructed.